apps/chat/src/utils/app/file.ts

```diff
@@ -152,7 +152,7 @@ export const getDialFilesFromAttachments = (
       const { absolutePath, name } = parseAttachmentUrl(attachment.url);
 
       return {
-        id: absolutePath,
+        id: constructPath(absolutePath, name),
```

Comment:
Just use `attachment.url` instead because it's absolutely equals  to `constructPath(absolutePath, name)`